build.rs

```diff
@@ -23,27 +23,22 @@
 
 extern crate cc;
 
-use std::io::{self, Write};
+use std::env;
 
 fn main() {
     // Check whether we can use 64-bit compilation
-    #[cfg(target_pointer_width = "64")]
-    let use_64bit_compilation = {
+    let use_64bit_compilation = if env::var("CARGO_CFG_TARGET_POINTER_WIDTH").as_ref().map(|s| s.as_str()) == Ok("64") {
```

Comment:
I think we want to fail if the env variable is undefined:
```suggestion
    let use_64bit_compilation = if env::var("CARGO_CFG_TARGET_POINTER_WIDTH").unwrap() == "64" {
```
is also simpler.